layouts/partials/download-list.hbs

```diff
@@ -1,5 +1,6 @@
 <section>
   <ul>
+    <li><a href="/{{site.locale}}/{{site.download.install-on-wsl.link}}/">{{site.download.install-on-wsl.text}}</a></li>
```

Comment:
Why is this going on the top, rather than just as part of the rest of the package manager links?

Reply:
Myles just filed an update to move it to the bottom. Wasn't aware of your protocol for adding new items. Now moved to the bottom of the list.

Not the purpose of this PR, but arguably it seems that a reordering for relevance would make this list more user-friendly to meet the most common use-case scenarios... 1) Installing Node.js via package manager, 2) Previous releases, 3) Install via WSL (?), etc.  
It can feel like a bit of a disconnect / point of confusion to look at that list and see "Signed SHASUMS.." as the first bullet. Anyone new to NodeJS or unfamiliar with SHASUMS may just stop reading as that seems very advanced / in-depth, whereas having "Installing via package manager" seems like it should be front and center.

Reply:
In regard to separate treatment from other package managers, this isn't just a call to install via package manager, it's an entirely different install location. The primary purpose is to highlight that NodeJS is a better overall dev experience on WSL using a Windows machine for most scenarios so want to raise awareness of it as an option.